compiler/luci/export/src/CircleExporterUtils.cpp

```diff
@@ -24,7 +24,19 @@
 
 namespace luci
 {
-
+circle::WeightCompressionType to_circle_weightcompressiontype(luci::WeightCompression type)
+{
+  switch (type)
+  {
+    case luci::WeightCompression::NONE:
+      return circle::WeightCompressionType_NONE;
+    case luci::WeightCompression::HUFFMAN:
+      return circle::WeightCompressionType_Huffman;
```

Comment:
I assume you are adding weight compression with Huffman coding.
As you adding this define here, I think `compress_weights_huffman` instead of general `compress_weights`, would be better name.

Reply:
@seanshpark fixed, please take a look